CMakeLists.txt

```diff
@@ -343,10 +344,6 @@ if(ENABLE_NETGEN)
     else()
         set_target_properties(NETGENPlugin PROPERTIES COMPILE_FLAGS "${NETGEN_CXX_FLAGS} -DNETGEN_VERSION=${NETGEN_VERSION}")
     endif()
```

Comment:
I guess this changes were intriduced by me and were more or less conda-specific. I guess removing them should not introduce troubles.

Reply:
I'm not sure what changes you're referring to here?

Reply:
I meant the 4 lines below.